### ISSUE TITLE
failing to upload my package with twine 

### ISSUE THREAD

**Issue by vstadnytskyi** [NSLS-II/scientific-python-cookiecutter#68]
I am having troubles with uploading my new package to Pypi.org.

Original terminal snippet and the error:
```
HTTPError: 400 Client Error: '0.0.0.post1.dev0+g83905ac' is an invalid value for Version. Error: Can't use PEP 440 local versions. See https://packaging.python.org/specifications/core-metadata for url: https://upload.pypi.org/legacy/
```

or I get this error

`HTTPError: 400 Client Error: 'X.Y.ZZ.post0.dev0-g83905ac' is an invalid value for Version. Error: Start and end with a letter or numeral containing only ASCII numeric and '.', '_' and '-'. See https://packaging.python.org/specifications/core-metadata for url: https://upload.pypi.org/legacy/`

**Comment by tacaswell** [comment]
What version of git are you using?

**Comment by mrakitin** [comment]
It should be resolved by https://github.com/vstadnytskyi/circular_buffer_091319/pull/1. @vstadnytskyi, is it still a problem?

**Comment by vstadnytskyi** [comment]
Yes, this is resolved. Here is a brief description of the problem. I have created a new repository following instructions, however, I didn't get the full set of files.I have noticed .travis.yml and other files were missing.  I don't know why. So I copied some of them from elsewhere. This created a mess which eventually was resolved by renaming the folder and running cookiecutter again and setting up everything from scratch. 

Which leads to this question:
It all could have been avoided if I listed all files including hidden files at the very beginning.
item 7 shows all non-hidden files:

```
$ ls example/
AUTHORS.rst        MANIFEST.in     example                 setup.cfg
CONTRIBUTING.rst   README.rst      requirements-dev.txt    setup.py
LICENSE            docs            requirements.txt        versioneer.py

```

 Item 8 in the Get Started instruction shows the tree, but it doesn't tell one how to see all hidden files. 

Maybe updating instructions to include list command with option all in item 7 or change item 8.

**8. Take a moment to see what we have. (can be accessed via` ls -a` command in the terminal window)**
